scripts/sysman/memory.yml

```diff
@@ -163,7 +163,8 @@ version: "1.7"
 class: $sMemory
 name: $s_mem_ext_bandwidth_t
 details:
-    - "Number of counter bits"
+    - "1.7": "Number of counter bits"
```

Comment:
so this actually was introduced in 1.7 and we are deprecating since we did not have proper stype. so do we still need to make this "1.0" ? or "1.7" is fine?

Reply:
@aravindksg this is already version 1.7 in line 162. No need for adding this versioning here.

Reply:
fixed